examples/bugs/gh-219/open-next.config.ts

```diff
@@ -1,25 +1,3 @@
-import type { OpenNextConfig } from "@opennextjs/aws/types/open-next.js";
+import { defineCloudflareConfig } from "@opennextjs/cloudflare/config";
 
-const config: OpenNextConfig = {
-  default: {
-    override: {
-      wrapper: "cloudflare-node",
-      converter: "edge",
-      // Unused implementation
-      incrementalCache: "dummy",
-      tagCache: "dummy",
-      queue: "dummy",
-    },
-  },
-
-  middleware: {
-    external: true,
-    override: {
-      wrapper: "cloudflare-edge",
-      converter: "edge",
-      proxyExternalRequest: "fetch",
-    },
-  },
-};
-
-export default config;
+export default defineCloudflareConfig();
```

Comment:
incrementalCache: "dummy" ?

Reply:
sorry I don't understand the question

Reply:
(`"dummy"` is not a valid value for this API)

Reply:
nevermind